rules/DowngradePhp73/Rector/ConstFetch/DowngradePhp73JsonConstRector.php

```diff
@@ -91,4 +125,118 @@ private function refactorIf(If_ $if): ?If_
 
         return $if;
     }
+
+    /**
+     * Add an alternative throwing error behavior after any `json_encode`
+     * or `json_decode` function called with the `JSON_THROW_ON_ERROR` flag set.
+     * This is a partial improvement of removing the `JSON_THROW_ON_ERROR` flag,
+     * only when the flags are directly set in the function call.
+     * If the flags are set from a variable, that would require a much more
+     * complex analysis to be 100% accurate, beyond Rector actual capabilities.
+     * @return null|array<StmtExpression|If_>
+     */
+    private function refactorStmt(StmtExpression $stmtExpression): ?array
+    {
+        $nodes = [$stmtExpression];
+
+        // retrieve a `FuncCall`, if any, from the statement
+        $funcCall = match ($stmtExpression->expr::class) {
+            Assign::class => $stmtExpression->expr->expr instanceof FuncCall
+                ? $stmtExpression->expr->expr
+                : null,
+            FuncCall::class => $stmtExpression->expr,
+            default => null,
+        };
```

Comment:
downgrade this part seems cause error: https://github.com/rectorphp/rector-src/actions/runs/5332525439/jobs/9664202507#step:15:72

Reply:
The error seems happen on php 7.4 https://3v4l.org/75qT2#v7.4.33

Reply:
I think it is because The scope of `$stmtExpression->expr` is already lost as parent re-printed, the better solution I think is refactor the code for now. I will create new PR.